deployment/docker-compose-local-basic.yml

```diff
@@ -34,6 +34,7 @@ services:
       - PROJECTSMO_DNA_BRAND_LOGO_URL=/images/branding/logo-brand.png
       - PROJECTSMO_DNA_APP_LOGO_URL=/images/branding/logo-app-white.png
       - PROJECTSMO_ENABLE_NOTIFICATION=true
+      - PROJECTSMO_INTERNAL_USER_TEAMS_INFO=(Recommended to use Short ID. To find Short ID use <a href="YOUR_TEAMS_INFO_URL" target="_blank" rel="noreferrer noopener">Teams</a>)
```

Comment:
please add escape sequence character

Reply:
- PROJECTSMO_STORAGE_TOU_HTML=<div>I agree to <a href=\"#\" target=\"_blank\" rel=\"noopener noreferrer\">terms of use</a></div>